marklogic-data-hub/src/main/resources/ml-modules/services/mlFlow.xqy

```diff
@@ -69,7 +72,7 @@ declare function get(
           return
             if (fn:exists($flow)) then $flow
             else
-              fn:error((),"RESTAPI-SRVEXERR", (404, "Not Found", "The requested flow was not found"))
+              httputils:throw-not-found((), "The requested flow was not found")
```

Comment:
Why include "()" as the first arg here?

Reply:
In many places we pass error name (first argument for e.g. in SM project), so included that param.